src/renderers/react/ElementWrapper.tsx

```diff
@@ -172,13 +175,15 @@ export const ElementWrapper: React.FunctionComponent<Props> = ({
           expanded
           disabled={!moveDown(false)}
           onClick={() => moveDown(true)}
+          aria-label="Move element down."
```

Comment:
The nittiest of nits – do aria elements need a full stop?

Reply:
Wise words - will remove!